### ISSUE TITLE
remove an image by force also remove the container which is using the image

### ISSUE THREAD

**Issue by gouyang** [projectatomic/buildah#438]
It remove all containers silently.

```
# buildah from busybox
busybox-working-container-1
# buildah from busybox
busybox-working-container-2
# buildah rmi -f busybox
untagged: docker.io/library/busybox:latest
5b0d59026729b68570d99bc4f3f7c31a2e4f2a5736435641565d93e7c25bd2c3
# buildah containers
#
```
docker is unable to remove the image if it's in use.
```
# docker rmi 84581e99d807
Error response from daemon: conflict: unable to delete 84581e99d807 (cannot be forced) - image is being used by running container d6cdb7497c31
```

**Comment by TomSweeneyRedHat** [comment]
Hi @gouyang,

Thanks for the report, but this is working as designed, when you use -f with 'buildah rmi' it removes the associated containers to and there's a warning in the man page.

```
--force, -f

This option will cause Buildah to remove all containers that are using the image before removing the image from the system.
```

**Comment by gouyang** [comment]
I just checked `buildah rmi -h`, not the man page. Please close the issue if no action need.

**Comment by TomSweeneyRedHat** [comment]
Thanks @gouyang!  I'll see if I can update the usage to make the help be clearer.

**Comment by rhatdan** [comment]
Yes lets make a one line change to the help to fix this.

**Comment by TomSweeneyRedHat** [comment]
Closed with #441, thanks for the issue report @gouyang
